### DISCUSSION TITLE
Application

### DISCUSSION THREAD

**Discussion by 01SOL** [iptv-org/community#150] [Q&A] [answered]
Is there a recommended application for iptv links on iOS ?

**Comment by Sphinxroot** [comment, marked as answer]
https://apps.apple.com/app/apple-store/id650377962  VLC

https://apps.apple.com/us/app/gse-smart-iptv/id1028734023

https://apps.apple.com/us/app/iptv-player-play-m3u-playlist/id1303327384

https://kodi.tv/download/ios + Playlistloader addon (no epg)
